packages/E2ETest/wdio/pages/TextBackgroundColorTestPage.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ */
+
+import { BasePage } from './BasePage';
+
+class TextBackgroundColorTestPage extends BasePage {}
+
+export default new TextBackgroundColorTestPage();
```

Comment:
Seems like you might be able to just use HomePage for this instead of adding another page, since we're not adding functionality